### Notebook Diffs vs origin/main
NOTEBOOK: newnew.ipynb

nbdiff /dev/fd/63 newnew.ipynb
--- /dev/fd/63  2025-07-30 14:41:48.095082
+++ newnew.ipynb  2025-07-30 14:41:23.256912
## inserted before /cells/1:
+  code cell:
+    source:
+      trigger cleanup




